test/e2e/storage/persistent_volumes-local.go

```diff
@@ -526,26 +549,42 @@ func setupLocalVolumeDirectory(config *localTestConfig) *localTestVolume {
 }
 
 // Deletes the PVC/PV, and launches a pod with hostpath volume to remove the test directory
-func cleanupLocalVolume(config *localTestConfig, volume *localTestVolume) {
-	if volume == nil {
-		return
+func cleanupLocalVolumeGCELocalSSD(config *localTestConfig, volume *localTestVolume) {
+	By("Cleaning up PVC and PV")
```

Comment:
Can you move the PVC/PV cleanup outside since it is common to all 3 volume types?